### PR TITLE
Add alt text to images issue#875

### LINKED ISSUE #391: [ACCESSIBILITY] - alt text on images cleanup
In the migrated documentation, we have a large number of missing, inaccurate, misleading, or non-conventional styles of alt text for accessibility purposes. This is obviously a very big task to dump into a single ticket at the moment, but at minimum we should:

- Start identifying the major sections of the guide which need serious cleanup, and create additional tickets if needed
- Make cleanup efforts where sections are small or clearly going to stick around for a while

### LINKED ISSUE #875: [ISSUE][DOCS] Image tags with empty and "Image NOT FOUND" alt text
## Describe the issue

There are many cases in the documentation where an image's alt text is empty `![]` or  `![Image NOT FOUND]`.  

## Impacted page(s)

Found 114 instances in 58 files.

## Expected experience

Alt text is related to the image.

### PR DESCRIPTION
**Pull request by FiniteStateGit**
Also added an image that was missing from https://o3de.org/docs/user-guide/engine/az-console/ to static.

Fix #875
Partial Fix #391

### CODE REVIEW

**Review by micronAMZN — APPROVED** [review]

**Comment by sptramer** [comment]
@FiniteStateGit Will approve and merge after the merge conflict is fixed.

**Review by sptramer — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**
